src/commands/commands.ts

```diff
@@ -685,6 +685,163 @@ const debugCmd = new SimpleAction(
   },
 );
 
+/** Label command, used to change the label of the given channel. */
+const labelCmd = new TwoPartCommand(
+  'label',
+  `Change the channel's label for easier debugging.`,
+  'label <bot name> <channel id> <channel label>',
+  // Group trigger
+  /^\s*label(?<group>.*)$/,
+  // Action trigger
+  /^\s*(?:(?<botName>\w+)\s+)?(?<channelID>(?:(?:\+|-)?\d+)|(?:this))\s+(?<channelLabel>.+?)\s*$/,
+  // Action
+  async (message, match) => {
+    const bot = message.getBot();
+    const { botName, channelID, channelLabel } = match.groups;
+
+    let channelBot = bot;
+    // Get the bot to find the channel on
+    if (botName) {
+      const findBot = getBots().find((client) => client.name === botName);
+
+      if (findBot) {
+        channelBot = findBot;
+      } else {
+        // Did not find the specified bot
+        const botList = getBots()
+          .map((client) => `- ${client.name}`)
+          .join('\n');
+
+        message.reply(
+          `'${botName}' is not an available bot! Try one of the following:\n${botList}`,
+        );
+      }
+    } else {
+      // No bot specified, take the bot that the message was written on
+      channelBot = bot;
+    }
+
+    let labelChannel = message.channel;
+
+    if (channelID === 'this') {
+      // Take this channel and this bot
+      labelChannel = message.channel;
+      channelBot = bot;
+    } else {
+      labelChannel = channelBot.getChannelByID(channelID);
+    }
+
+    let label = channelLabel.trim();
+
+    // Check if the user wants to reset the label
+    if (label === 'reset') {
+      label = undefined;
+      bot.sendMessage(
+        message.channel,
+        `Removing the label of channel ${labelChannel.id} on ${channelBot.name}.`,
+      );
+    } else {
+      bot.sendMessage(
+        message.channel,
+        `Changing the label of channel ${labelChannel.id} on ${channelBot.name} to '${label}'.`,
+      );
+    }
+
+    // Save locally
+    labelChannel.label = label;
+
+    // Save in the JSON file
+    const subscribers = DataManager.getSubscriberData();
+    const channels = subscribers[channelBot.name];
+
+    // Check if the channel is already registered
+    const existingChannelId = channels.findIndex((ch) => labelChannel.isEqual(ch.id));
+    if (existingChannelId >= 0) {
+      const existingChannel = channels[existingChannelId];
+      // Update label
+      existingChannel.label = label;
+
+      // Remove unnecessary entries
+      if (
+        existingChannel.gameSubs.length === 0 &&
+        !existingChannel.prefix &&
+        !existingChannel.label
+      ) {
+        channelBot.logger.debug('Removing unnecessary channel entry...');
```

Comment:
This log could also use a channel label/id

Reply:
Added.